openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -58,11 +61,10 @@ OpenAssessment.ResponseView.prototype = {
                 view.server.renderLatex($(stepID, view.element));
                 view.installHandlers();
                 view.setAutoSaveEnabled(true);
-
                 view.isRendering = false;
                 view.baseView.announceStatusChangeToSRandFocus(stepID, usageID, false, view, focusID);
                 view.announceStatus = false;
-
+                view.dateFactory.report();
             }
         ).fail(function() {
```

Comment:
These calls seem symptomatic of an underlying issue.  Is there no common place where we can activate the dateFactory code?